hack/doc/README.md

```diff
@@ -0,0 +1,26 @@
+# role-permissions2md - YAML to Markdown converter for operator permissions
+
+This script can be used to create a table containing permissions needed by components of the Dynatrace Kubernetes Operator. The resulting tables can be used as input for SUS tickets to update our public documentation here: https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-container-platforms/kubernetes/get-started-with-kubernetes-monitoring/dt-component-permissions#dto
+
+Pre-requisites:
+```
+python3
+pip install pyyaml
+```
+
+## Usage
+
+It's best to use the `openshift-all.yaml` manifest to cover all our needs. OpenShift needs the most permissions.
+
+```
+# local dev repo - direct call
+make manifests
+python3 <operator-repo>/hack/doc/role-permissions2md.py <operator-repo>/config/deploy/openshift/openshift-all.yaml
+
+# local dev repo - using make, result will be locally in permissions.md
+make doc/permissions
```

Comment:
it's enough to document the `make` command and if there's another step required to use them from the web, it should be something that can be passed into the `make` step via env var or similar.

Reply:
I like to have a somewhat complete documentation of what the script in it's readme. If in a situation to find out, how a script works, I find it tedious having to dig around in make files.